### DISCUSSION TITLE
Want to contribute

### DISCUSSION THREAD

**Discussion by bicrxm** [ACINQ/eclair#2187] [Developers]
Hi everyone. I have come across this project through summerofbitcoin. I want to contribute/participate in this project. Can I know how to get started?

**Comment by t-bast** [comment]
Hi! That depends, what do you want to do? Any particular feature you're interested in?
Otherwise I'd suggest to go through the issues and pick one that is marked "good first issue" or "help wanted" and try to fix that.

**Comment by barrybecker4** [comment]
I also would like to learn and contribute. I checked out locally and noticed that [one of the tests failed in Intellij](https://github.com/ACINQ/eclair/issues/2288). I created a [PR](https://github.com/ACINQ/eclair/pull/2289) to fix it, but it says
![image](https://user-images.githubusercontent.com/3020163/170869503-9cacd047-5ad5-4962-9f25-cf338f0a0d97.png)
Can I get a quick review and approval for the PR build?
